misc/scripts/update.sh

```diff
@@ -36,8 +36,11 @@ sudo chown "$PACSTALL_USER" -R /tmp/pacstall
 sudo mkdir -p /usr/share/bash-completion/completions
 
 STGDIR="/usr/share/pacstall"
+tabs -4
 
 tty_settings=$(stty -g)
+old_pacstall_version=( $(pacstall -V) )
+old_pacstall_branch=( $(cat $STGDIR/repo/update) )
```

Comment:
Misleading variable name. This contains both the repository user, and the branch name.  Split the array into 2 individual variables.

Do the same for `new_pacstall_branch`